scripts/docker/hub/Dockerfile

```diff
@@ -10,27 +10,23 @@ RUN apt update && apt install -y --no-install-recommends openssl libudev-dev fil
 # show backtraces
 ENV RUST_BACKTRACE 1
 
-#cleanup Docker image
-RUN apt autoremove -y
-RUN apt clean -y
-RUN rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
+# cleanup Docker image
+RUN apt autoremove -y \
+  && apt clean -y \
+  && rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
 
 RUN groupadd -g 1000 parity \
-  && useradd -m -u 1000 -g parity -s /bin/sh parity
+  && useradd -m -u 1000 -g parity -s /bin/bash parity
 
-
-#add TARGET to docker image
+# add TARGET to docker image
 COPY artifacts/x86_64-unknown-linux-gnu/$TARGET /bin/$TARGET
 
-# Build a shell script because the ENTRYPOINT command doesn't like using ENV
-RUN echo "#!/bin/bash \n ${TARGET} \$@" > ./entrypoint.sh
-RUN chmod +x ./entrypoint.sh
-
 COPY scripts/docker/hub/check_sync.sh /check_sync.sh
 
 # switch to user parity here
 USER parity
 
-# setup ENTRYPOINT
+VOLUME [ "/home/parity/.local" ]
```

Comment:
Might be better to be more precise `/home/parity/.local/share/io.parity.ethereum`.